Presentacion/public_html/js/views/DatosBasicos/DatosBasicosView.js

```diff
@@ -16,24 +16,24 @@ define([
             if(id){
                 this.model.set({idProfesor:id});
             }
+            var that = this;
             var options = {
-                success:this.cargarDatos,
+                success: function(pmo){
+                    that.cargarTemplate({model:pmo       },ListarDatosBasicosTemplate);
+                },
                 error:this.errorConsulta
             };
-            //this.model.sync("read",this.model,options);
-            this.cargarDatos(this.model);
+            this.model.sync("read",this.model,options);
+            //this.cargarDatos(this.model);
+        },
+        cargarEditar: function(){
+            this.cargarTemplate({model:this.model},EditarDatosBasicosTemplate);
```

Comment:
Change to `this.cargarTemplate({model: this.model}, EditarDatosBasicosTemplate);`